packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -203,6 +192,29 @@ export function trapEventForPluginEventSystem(
   } else {
     fbListener = addEventBubbleListener(container, rawEventName, listener);
   }
+  // If we have an fbListener, then use that.
+  // We'll only have one if we use the forked
+  // EventListener-www module in FB builds.
+  return fbListener || listener;
+}
+
+export function removeTrappedPassiveEventListener(
+  document: Document,
+  topLevelType: string,
+  listener: any => void,
+) {
+  if (listener.remove != null) {
```

Comment:
This is not great because we're shipping this code even if it's not an FB listener. Like if Function.prototype.remove is added on the page. It also confuses the compiler. We should be sure which path we're using and consistently call that - statically